### PR TITLE
#249 footer 2 version for tablet

### PR DESCRIPTION
**Pull request by Mikivitkovski**

### CODE REVIEW

File: components/module-components/contact/ContactForm.tsx [hunk 1]
```diff
@@ -78,6 +79,7 @@ const ContactForm = () => {
           type="email"
           field="email"
           formik={formik}
+          inputClass={['']}
```
Comment (edichoska) [inline components/module-components/contact/ContactForm.tsx:82]: can't this property be optional? so we don't have to pass empty inputClass

File: components/reusable-components/button/button.module.scss [hunk 2]
```diff
@@ -20,6 +20,10 @@
   }
 }
 
+.smallFooterButton {
```
Comment (edichoska) [inline components/reusable-components/button/button.module.scss:23]: better class naming here

File: components/reusable-components/text-input/textInput.module.scss [hunk 3]
```diff
@@ -35,6 +36,33 @@
       }
     }
 
+    .newsletterInput {
+      border: 1px solid white;
+      background-color: transparent;
+      color: white !important;
+      height: 3.375rem;
+      -webkit-text-fill-color: #ffff !important;
+      -webkit-text-color: #ffff !important;
```
Comment (edichoska) [inline components/reusable-components/text-input/textInput.module.scss:45]: can we use the white color defined in colors

File: components/reusable-components/text-input/textInput.module.scss [hunk 4]
```diff
@@ -35,6 +36,33 @@
...
+    }
+
+    .lightNewsletterInput {
+      -webkit-box-shadow: inset 0 0 0 30px #121d36 !important;
+      background-clip: content-box !important;
+    }
+
+    .darkNewsletterInput {
+      -webkit-box-shadow: inset 0 0 0 30px #070b15 !important;
+      background-clip: content-box !important;
```
Comment (edichoska) [inline components/reusable-components/text-input/textInput.module.scss:63]: also these colors here, if we don't have them in colors can you please define them there so we can reuse them

**Review by edichoska — CHANGES_REQUESTED** [review]
@Mikivitkovski Miki this looks good. I think this version of the footer is looking much better. We have just some small details to look into it.

File: components/reusable-components/button/button.module.scss (hunk 2 above)
Comment (Mikivitkovski) [reply components/reusable-components/button/button.module.scss:23]: i didnt find where is used this and i delete it

**Review by edichoska — APPROVED** [review]
🚀

**Pull request merged**